desktop/core/src/desktop/management/commands/config_override.py

```diff
@@ -0,0 +1,62 @@
+from configobj import ConfigObj
```

Comment:
Could you add a header/license section like https://github.com/cloudera/hue/blob/master/apps/beeswax/src/beeswax/management/commands/beeswax_install_examples.py#L1 ?